README.md

```diff
@@ -10,7 +10,7 @@ Docs available [here](https://fastexcel.toucantoco.dev/).
 
 ### Prerequisites
 
-Python>=3.10 and a recent Rust toolchain must be installed on your machine. `cargo` must be available in your `PATH`.
+Python>=3.8 and a recent Rust toolchain must be installed on your machine. `cargo` must be available in your `PATH`.
```

Comment:
We were planning on publishing a crate for fastexcel at some point. Would you be interested in an excel reader in polars' Rust API ? If so, what is your MSRV ?

Reply:
Longer term we were thinking of writing our own `calamine` bindings directly so that we can get the lowest possible level of integration/customisation (we've now forked `arrow2` and brought it inside as `polars-arrow`) but there's no timeframe on that yet, so... maybe? :)  Our current toolchain is `nightly-2023-12-23`.

Reply:
All right, I'll let you know if we start working on this :)